AK/String.cpp

```diff
@@ -484,7 +484,7 @@ ErrorOr<String> String::trim(StringView code_points_to_trim, TrimMode mode) cons
     return trim(Utf8View { code_points_to_trim }, mode);
 }
 
-ErrorOr<String> String::trim_whitespace(TrimMode mode) const
+ErrorOr<String> String::trim_ascii_whitespace(TrimMode mode) const
```

Comment:
Don't rename it in the Chess commit, add the method with the correct name int the first place